docs/product-getting-started-guide/src/main/asciidoc/decision-table-create-proc.adoc

```diff
@@ -0,0 +1,107 @@
+
+[float]
+== Prerequisite
+
+Created both the Violation and Driver data objects.
+
+.Procedure
+. Log in to {CENTRAL}. Click *Menu* -> *Design* -> *Projects*, then *Driver department traffic violations*.
+. Click *Create New Asset* -> *Guided Decision Table*, then enter:
+
+* *Guided Decision Table*: `SpeedViolationRule`
+* *Package*: `org.jboss.example.traffic_violations`
++
+
+. Select *Unique Hit* from the *Hit Policy* pull-down menu.
+. Select *Extended entry, values defined in table body* in the *Table format* section.
+. Click *Ok* to open the *Guided Decision Tables* designer.
++
+
+.Guided Decision Tables designer
+image::dec-table.png[]
+
+== Inserting Violation columns
+
+. Click *Columns* -> *Insert Column* and select *Include advanced options*.
+. Select *Add an Attribute column* and click *Next*.
++
+
+.Add a new column window
+image::new-column.png[]
+. Select *Ruleflow-group* and click *Finish*.
+. Expand *Attribute columns* and enter `trafficViolation` in the *Default value* field.
++
+
+.Attribute columns window
+image::attrib-column.png[]
+. Click *Insert Column*, select *Add a Condition* -> *Pattern* -> *+Create a new Fact Pattern*.
+. Select *Violation* from the *Fact type* pull-down menu, enter `v` in the *Binding* field, and click *+OK*.
++
+
+.Create a new fact pattern window
+image::fact-pattern.png[]
+. Select *Calculation type* -> *Literal value*.
+. Select *Field* and then *type* from the *Field* pull-down menu.
+. Select *Operator* and then *equal to* from the *Operator* pull-down menu.
+. Select *Value options* and enter `Driving while intoxicated,DWI=Driving while under the influence of drugs` in the *Value list (optional)* field.
+. Select *Additional info*, enter `Violation Type` in the *Header (description)* field, and click *Finish*.
++
+
+.Violation Type header
+image::violation-columns.png[]
+. Click *Insert Column*, select *Add a Condition* -> *Pattern* and select *Violation[v]* from the *Pattern* pull-down menu.
+. Select *Calculation type* -> *Predicate* -> *Field* and enter `actualSpeed-speedLimit > $param`.
+. Select *Value options*, then select *Additional info* and enter `Speed Limit (MPH) >` in the *Header (description)* field.
+. Click *Finish*.
++
+
+.Speed Limit (MPH) > header
+image::speed-header.png[]
+. Click *Insert Column*, select *Add a Condition* -> *Pattern* and select *Violation[v]* from the *Pattern* pull-down menu.
+. Select *Calculation type* -> *Predicate*.
+. Select *Field* and enter `actualSpeed-speedLimit < $param` in the *Field* field.
+. Select *Operator*, select *Value options*, then select *Additional info*.
+. Enter `Speed Limit (MPH) <` in the *Header (description)* field and click *Finish*.
++
+
+.Speed Limit (MPH) < header
+image::speed-less-header.png[]
+
+== Inserting Result columns
+
+. Click *Insert Column*, select *Set the value of a field* -> *Pattern* and select *Violation[v]* from the *Pattern* pull-down menu.
```

Comment:
Serial comma again. It comes up elsewhere. I won't mention it again, just in series, per IBM/Chicago/Red Hat, we us a serial comma. Only AP style really doesn't do that. Been working for the press much? jk :)

Reply:
eagle eye..